Adds tokengating use case to NFT page (#8254)
ethereum/ethereum-org-website: src/content/nft/index.md

<Divider />

### Tokengating {#tokengating}

Tokengating is a way of restricting access to something and using tokens as a way to unlock access. These things can vary hugely based on the platform but popular examples are gated content, private chat servers, and, in the world of ecommerce, exclusive products.   

> A platform that does tokengating will typically ask you to connect your wallet to prove you own the required tokens. If you have the tokens you need, you'll get access. If not, the thing will remain gated. NFTs are a great way to do this because of their uniqueness – you can't fake ownership to get the thing. 

Reviewer: ```suggestion
A platform that does tokengating will typically ask you to connect your wallet to prove you own the required NFT. If you have the NFT you need, you'll get access. If not, the thing will remain gated. NFTs are a great way to do this because of their uniqueness – you can't fake ownership to get the thing. 
```

> And because NFTs are on Ethereum, your NFTs can unlock Ethereum tokengates on any platform that implements them. The one NFT you own could unlock the gated content, private chat servers, and exclusive products across completely different websites and applications.

Reviewer: ```suggestion
Since NFTs are on Ethereum, they can be used to unlock Ethereum tokengates on any platform that implements them. A single NFT you own could unlock gated content, private chat servers, and exclusive products across completely different websites and applications.
```
